test/transport/request.js

```diff
@@ -251,5 +252,23 @@ describe('Request HTTP transport', () => {
         })
         .catch(assert.ifError);
     });
+
+    it('enables uses verbatim', () => {
+      nock.cleanAll();
+      api.get('/').reply(200, simpleResponseBody);
+
+      sinon.spy(dns, 'lookup');
+
+      const ctx = createContext(url);
+
+      return new RequestTransport()
+        .execute(ctx)
+        .then((ctx) => {
+          ctx.res.httpResponse.request.agentOptions.lookup('www.example.com', {}, () => {});
```

Comment:
This isn't a brilliant way to test that `dns.lookup()` is called with `verbatim: true` set. Open to any suggestions on a better way to do this.